install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -431,12 +431,12 @@ data:
 {{- if .Values.bpf.natMax }}
   # bpf-nat-global-max specified the maximum number of entries in the
   # BPF NAT table.
-  bpf-nat-global-max: "{{ .Values.bpf.natMax | int }}"
+  bpf-nat-global-max: "{{ .Values.bpf.natMax | int | quote }}"
 {{- end }}
 {{- if .Values.bpf.neighMax }}
   # bpf-neigh-global-max specified the maximum number of entries in the
   # BPF neighbor table.
-  bpf-neigh-global-max: "{{ .Values.bpf.neighMax | int }}"
+  bpf-neigh-global-max: "{{ .Values.bpf.neighMax | int | quote }}"
```

Comment:
Same as above

Reply:
Fixed in https://github.com/cilium/cilium/commit/4708b740f6f60f2671187d1b304f3dd5ba07787d, thanks!